OSDOCS-4328: Edits from the Optimizing Routing content audit (#56929)
openshift/openshift-docs: modules/baseline-router-performance.adoc

[id="baseline-router-performance_{context}"]
= Baseline Ingress Controller (router) performance

> The {product-title} Ingress Controller, or router, is the ingress point for ingress traffic for applications and services that are configured using routes and ingresses.

Reviewer: ```suggestion
The {product-title} Ingress Controller provides a means for traffic external to the cluster to access services internal to the cluster. When routing HTTP or HTTPS traffic, this is also called an HAProxy router."
```

Reviewer 2: No, this is not correct.

> When evaluating a single HAProxy router performance in terms of HTTP requests handled per second, the performance varies depending on many factors. In particular:

Reviewer: ```suggestion
When evaluating a single HAProxy router performance in terms of HTTP requests handled per second, the performance varies according to the following factors:
```

Reviewer 2: No, that changes the meaning.

|re-encrypt |2320|2941
|===

> The default Ingress Controller configuration was used with the `spec.tuningOptions.threadCount` field set to `4`. Two different endpoint publishing strategies were tested, Load Balancer Service and Host Network. TLS session resumption was used for encrypted routes. With HTTP keep-alive, a single HAProxy router is capable of saturating 1 Gbit NIC at page sizes as small as 8 kB.

Reviewer: ```suggestion
The default Ingress Controller configuration was used with the `spec.tuningOptions.threadCount` field set to `4`. Two different endpoint publishing strategies were tested: Load Balancer Service and Host Network. TLS session resumption was used for encrypted routes. With HTTP keep-alive, a single HAProxy router is capable of saturating a 1 Gbit NIC at page sizes as small as 8 kB.
```

|===

> In general, HAProxy can support routes for 5 to 1000 applications, depending on the technology in use. Ingress Controller performance might be limited by the

Reviewer 2: There is really no lower bound on the number of applications, so you could say "In general, HAProxy can support routes for up to 1000 applications, depending on..."
